sklearn/neural_network/_multilayer_perceptron.py

```diff
@@ -1263,6 +1285,24 @@ class MLPRegressor(RegressorMixin, BaseMultilayerPerceptron):
     out_activation_ : string
         Name of the output activation function.
 
+    Examples
+    ----------
+    >>> from sklearn.neural_network import MLPRegressor
+    >>> from sklearn.datasets import make_regression
+    >>> from sklearn.model_selection import cross_val_score
+
+    >>> X, y = make_regression(n_samples=20, n_features=5, random_state=1)
+
+    >>> clf = MLPRegressor(random_state=1).fit(X, y)
+    >>> # Predict values
+    >>> print(clf.predict([[0.1,0.69,0.54,0.1,0.27]]))
+    [12.61233323]
+
+
+    >>> # Calculate coefficient of determination R^2
+    >>> cross_val_score(clf, X, y, cv=5)
```

Comment:
Maybe just `clf.score(X, y)` I'm not sure that doing cross validation with 20 samples is ideal anyway.